js/app.js

```diff
@@ -597,9 +601,34 @@
 				}
 			} else {
 				OCA.SpreedMe.app.videoWasEnabledAtLeastOnce = false;
-				OCA.SpreedMe.app.disableVideo();
+				// OCA.SpreedMe.app.availableAudio();
+				OCA.SpreedMe.app.availableVideo();
 			}
 		},
+		//// INFO: Not needed: no microphone results in an access denied failure
```

Comment:
And here. Also what happens if a user has video but no audio?

Reply:
OK, the comments will be removed. If there is Video but no Audio (microphone) the session wont be initialized! This is a bug, a issue is opened!